libafl/src/stages/calibrate.rs

```diff
@@ -220,25 +226,20 @@ where
             i += 1;
         }
 
+        let mut send_default_stability = false;
         let unstable_found = !unstable_entries.is_empty();
         if unstable_found {
+            let metadata = state.metadata_or_insert_with(UnstableEntriesMetadata::new);
```

Comment:
you can construct this without `()`? ( i didn't know

Reply:
The function takes a fn without parameter and calls it, new is a fn without parameters, so I can pass new directly. It's then being called inside (in case it doesn't already exists in the map)

Reply:
It doesn't work in general/other contexts